lib/ansible/modules/source_control/git_config.py

```diff
@@ -157,9 +176,10 @@ def main():
             name=dict(type='str'),
             repo=dict(type='path'),
             scope=dict(required=False, type='str', choices=['local', 'global', 'system']),
-            value=dict(required=False)
+            value=dict(required=False),
+            state=dict(required=False, choices=['present', 'absent'])
```

Comment:
As above, all of the parameters in this list are in alphabetical order except `state`.

Reply:
Understood, will rearrange in the next commit. Thanks very much for the feedback.

Reply:
No problem. Congrats on your first Github PR :)